Financial-Times/n-myft-dropdown
# UG-879 Implement Tracking spec

### Description
- Added the `data-trackable` properties for the CTA events
- Added a demo page

### Ticket
https://financialtimes.atlassian.net/browse/UG-879

### How do I test this code?
- Check out this branch locally
- Run `npm run build` and `npm run demo` to start the demo page
- Open the demo page on `http://localhost:3000/` and confirm that the events defined on the [tracking spec](https://docs.google.com/spreadsheets/d/1XTIamP8h_6jKR0hwKlfSsdakb_IB1TvUrn214bhwLsg/edit#gid=0) are sent with the ir context

### Reminder
Have you completed these common tasks? (remove those that don't apply)


### How we will review this PR
Our team uses [conventional comments](https://conventionalcomments.org/) to provide feedback.
